src/frontend/Komponenter/Journalforing/LeggTilBarnSomSkalFødes.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { FamilieDatovelger } from '@navikt/familie-form-elements';
+import LeggtilMedSirkel from '../../Felles/Ikoner/LeggtilMedSirkel';
+import SlettSøppelkasse from '../../Felles/Ikoner/SlettSøppelkasse';
+import { BarnSomSkalFødes } from '../../App/hooks/useJournalføringState';
+import { Button, Heading } from '@navikt/ds-react';
+import styled from 'styled-components';
+import navFarger from 'nav-frontend-core';
+
+const Tittel = styled(Heading)`
+    color: ${navFarger.navBlaLighten40};
+`;
+
+const LeggTilBarnContent = styled.div`
+    margin: 0.5rem 0;
+`;
+
+const InlineContent = styled.div`
+    border-left: 2px solid ${navFarger.navBlaLighten40};
+    padding-left: 1rem;
+    margin-left: 1rem;
+`;
+
+const TerminbarnMedDatovelger = styled.div`
+    display: grid;
+    grid-auto-columns: minmax(auto, 12rem);
+    grid-auto-flow: column;
+
+    > * {
+        align-self: flex-end;
+        padding: 0.5rem 0.5rem;
+    }
+`;
+
+const FjernBanrKnapp = styled(Button)`
+    width: 4rem;
+`;
+const LeggTilBarnKnapp = styled(Button)`
+    margin-top: 1rem;
+`;
+
+const LeggTilBarnSomSkalFødes: React.FC<{
+    barnSomSkalFødes: BarnSomSkalFødes[];
+    oppdaterBarnSomSkalFødes: (terminbarn: BarnSomSkalFødes[]) => void;
+}> = ({ barnSomSkalFødes, oppdaterBarnSomSkalFødes }) => {
+    const leggTilBarn = () => oppdaterBarnSomSkalFødes([...barnSomSkalFødes, {}]);
+
+    const fjernBarn = (terminbarnSomSkalSlettes: BarnSomSkalFødes) =>
+        oppdaterBarnSomSkalFødes(
+            barnSomSkalFødes.filter((barn) => barn !== terminbarnSomSkalSlettes)
+        );
+
+    const oppdaterTermindato = (terminbarnSomSkalOppdateres: BarnSomSkalFødes, dato: string) =>
+        oppdaterBarnSomSkalFødes(
+            barnSomSkalFødes.map((barn) =>
+                barn === terminbarnSomSkalOppdateres ? { fødselTerminDato: dato as string } : barn
+            )
+        );
+
+    return (
+        <LeggTilBarnContent>
+            <Tittel spacing size="xsmall" level="6">
+                Journalføre papirsøknad?
+            </Tittel>
+            <InlineContent>
+                Noe tekst
+                {barnSomSkalFødes.map((barn, index) => (
+                    <TerminbarnMedDatovelger key={index}>
```

Comment:
Vurder om vi skal ha en uuid på terminbarna og bruke den istedenfor indeks

Reply:
Ja, det jeg lurte på egentlige. Ville trott att dette ikke skulle gå men har testet og virker. Men er vel kanskje bedre å sette et `_id` som kun brukes for frontend for å ikke blande inn indeks i key